### DISCUSSION TITLE
Not receing emails after installation

### DISCUSSION THREAD

**Discussion by GenesisAdam** [simple-login/app#1253] [Self-hosting question]
I installed SL and received the Welcome email and successfully verified my email.
I am unable to send email using the reverse email address or receive emails from the aliases.
I have gone through the troubleshooting tips but all is Ok.
Looking into the sl-email and sl-postfix logs, I see the following but can't understand what is causing them:

**alias root@ebony4life.com cannot be created on-the-fly, return 550

error: unsupported dictionary type: btree

status=bounced (host sl-email[240.0.0.6] said: 550 SL E3 Email not exist (in reply to end of DATA command))**

Any help will be greatly appreciated.

**Comment by nguyenkims** [comment]
I'd suggest to run through the troubleshooting guide at https://github.com/simple-login/app/blob/046748c443dd42df6e8d7772b4e651cb77955231/docs/troubleshooting.md#L1-L0 as email setup is quite error prone and it's hard to know if one step in the setup has been missed.

**Comment by GenesisAdam** [comment]
I have tried those troubleshooting guides before asking for help. 
I am receiving all the welcome emails and telnet test was successful. The tests of the postfix were also successful without any errors or failure.
The only observation I see is that my gpg and upload folders are empty and I don't know if that is expected. I have followed the installation guides more than once but keep getting the same result. I use the version 3.4.0 which is the last official self-host release.
I followed the installation guide here: https://github.com/FarisZR/app by FarisZR.

**Reply by nguyenkims** [reply]
There seems to be an issue with the DKIM signing library as mentioned in https://github.com/simple-login/app/issues/1203, I wonder if you can try to disable DKIM signing and see if the issue is solved? DKIM can be disabled if `DKIM_PRIVATE_KEY_PATH` is removed from the config file.
